quickstarts/install-netobserv.yaml

```diff
@@ -0,0 +1,62 @@
+        1. In the OpenShift Container Platform web console, click **OperatorHub** from the [Operators]{{highlight qs-nav-operators}} → OperatorHub.
+        2. Choose **Network Observability Operator** from the list of available Operators in the OperatorHub, and click **Install**.
+        3. Select the checkbox **Enable Operator recommended cluster monitoring on this Namespace**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |-
+          #### Verify the operator was successfully installed:
+          Navigate to **Operators** → **Installed Operators**.
+          Is the Network Observability Operator listed?
+      summary:
+        failed: Try the steps again.
+        success: Great work! You successfully installed the Network Observability Operator.
+      title: Install the Network Observability Operator
+    - description: |-
+        In order to start capturing traffic, you need to create a **FlowCollector** resource to configure the components of Network Observability Operator.
+
+        1. Navigate to Operators → Installed Operators and select **Network Observability Operator**.
+
+        2. Navigate to the **FlowCollector** tab, and click **Create FlowCollector**. Make the following selections in the form view:
+          
+          2.1 **spec.agent.ebpf.Sampling**: Specify a sampling size for flows. Lower sampling sizes will have higher impact on resource utilization. For more information, see the "FlowCollector API reference", spec.agent.ebpf.
+          
+          2.2 If you are not using Loki, click Loki client settings and change **Enable** to **False**. The setting is True by default.
+          
+          2.3 If you are using Loki, set the following specifications:
+          
+            i. **spec.loki.mode**: Set this to the **LokiStack** mode, which automatically sets URLs, TLS, cluster roles and a cluster role binding, as well as the authToken value. Alternatively, the Manual mode allows more control over configuration of these settings.
+          
+            ii. **spec.loki.lokistack.name**: Set this to the name of your LokiStack resource. In this documentation, loki is used.
+          
+          2.4 Optional: If you are in a large-scale environment, consider configuring the FlowCollector with **Kafka** for forwarding data in a more resilient, scalable way. See "Configuring the Flow Collector resource with Kafka storage" in the "Important Flow Collector configuration considerations" section.
+          
+          2.5 Optional: Configure other optional settings before the next step of creating the FlowCollector. For example, if you choose not to use Loki, then you can configure exporting flows to **Kafka**, **OpenTelemetry** or **IPFIX**. See "Export enriched network flow data to Kafka and IPFIX" and more in the "Important Flow Collector configuration considerations" section.
```

Comment:
When saying "Configure other optional settings before the next step", it might be wrongly interpreted as "you absolutely have to do it NOW you can't do it later" ... so maybe clarify that it can _also_ be changed later?

Reply:
Maybe we should get rid of it in this quickstart ?
The goal is to step into network observability. Not covering all the cases here.

```suggestion
```

Reply:
yeah sgtm